src/build/SharedDeps.zig

```diff
@@ -411,11 +411,18 @@ pub fn add(
     })) |dep| {
         step.root_module.addImport("z2d", dep.module("z2d"));
     }
-    if (b.lazyDependency("ziglyph", .{
+    if (b.lazyDependency("zg", .{
         .target = target,
         .optimize = optimize,
     })) |dep| {
-        step.root_module.addImport("ziglyph", dep.module("ziglyph"));
+        if (self.config.emit_bench or self.config.emit_unicode_test) {
+            step.root_module.addImport("Graphemes", dep.module("Graphemes"));
+            step.root_module.addImport("DisplayWidth", dep.module("DisplayWidth"));
+        }
```

Comment:
I tried moving these addImports into GhosttyBench and GhosttyUnicodeTest, but it didn't work for GhosttyBench for some reason.